lua/lspconfig/spectral.lua

```diff
@@ -0,0 +1,32 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = 'spectral'
```

Comment:
I've been enforcing a convention recently where x-language-server type names get postfixed with _ls (spectral_ls), up for discussion though

Reply:
It's good to me, put the convention on the contrib file. 
I'll change that.